test/testcafe/spec/EnrollAuthenticatorEmail_spec.js

```diff
@@ -293,20 +302,20 @@ test
 test
   .requestHooks(logger, validOTPmock)('resend after 30 seconds', async t => {
     const enrollEmailPageObject = await setup(t);
-    await t.expect(enrollEmailPageObject.resendEmail.isHidden()).ok();
+    await t.expect(await enrollEmailPageObject.resendEmailExists()).eql(false);
     await t.wait(31000);
-    await t.expect(enrollEmailPageObject.resendEmail.isHidden()).notOk();
-    await t.expect(enrollEmailPageObject.resendEmail.getText()).eql('Haven\'t received an email? Send again');
+    await t.expect(await enrollEmailPageObject.resendEmailExists()).eql(true);
+    await t.expect(enrollEmailPageObject.resendEmailText()).contains('Haven\'t received an email?');
 
     // 8 poll requests in 31 seconds and 1 resend request after click.
     await t.expect(logger.count(
       record => record.response.statusCode === 200 &&
         record.request.url.match(/poll/)
-    )).eql(8);
+    )).gte(8);
```

Comment:
Are we polling more than 8 times in v3? If so, how many more times?

Reply:
I did see it fail on odd occasions where v3 was polling > 8 times. I tested locally now and it seems to be passing with `.eql(8)` assertion. Will test a few more times to make sure it's stable and push the change.

Reply:
Changed it back to `.eql(8)` assertion. Will monitor the CI runs to make sure it's green. So far, it's stable locally.